lib/mayaUsd/fileio/jobContextRegistry.cpp

```diff
@@ -82,10 +82,9 @@ void UsdMayaJobContextRegistry::RegisterExportJobContext(
                     itFound->niceName.GetText());
             }
             // Fill the export part:
-            ContextInfo updatedInfo {
-                key,        itFound->niceName,          TfToken(description),
-                enablerFct, itFound->importDescription, itFound->importEnablerCallback
-            };
+            ContextInfo updatedInfo(*itFound);
```

Comment:
Old code was losing the import context UI callback. New code preserve all already registered info automatically.